config/terraform.spc

```diff
@@ -21,4 +22,5 @@ connection "terraform" {
   # Defaults to CWD
   configuration_file_paths = ["*.tf"]
   plan_file_paths          = ["tfplan.json", "*.tfplan.json"]
+  state_file_paths         = ["terraform.tfstate", "*.tfstate"]
```

Comment:
I don't think we need `terraform.tfstate` if we have `*.tfstate`, since this includes it already

Reply:
updated.